libs/remix-ui/run-tab/src/lib/css/run-tab.css

```diff
@@ -489,6 +489,7 @@
   overflow: hidden;
   text-overflow: ellipsis;
   font-size: 11px;
+  width: 261px;
```

Comment:
Can you please explain your reasoning for picking these many units? why 261px? Magic numbers and strings make code brittle.